packages/react-core/src/components/LoginPage/examples/LoginPageLanguageSelect.tsx

```diff
@@ -0,0 +1,184 @@
+import React from 'react';
+import brandImg2 from './brandImgColor2.svg';
+import {
+  LoginFooterItem,
+  LoginForm,
+  LoginMainFooterBandItem,
+  LoginMainFooterLinksItem,
+  LoginPage,
+  ListItem,
+  ListVariant,
+  Select,
+  SelectOption
+} from '@patternfly/react-core';
+import ExclamationCircleIcon from '@patternfly/react-icons/dist/esm/icons/exclamation-circle-icon';
+
+export const LoginPageLanguageSelect: React.FunctionComponent = () => {
+  const [showHelperText, setShowHelperText] = React.useState<boolean>(false);
+  const [username, setUsername] = React.useState<string>('');
+  const [isValidUsername, setIsValidUsername] = React.useState<boolean>(true);
+  const [password, setPassword] = React.useState<string>('');
+  const [isValidPassword, setIsValidPassword] = React.useState<boolean>(true);
+  const [isRememberMeChecked, setIsRememberMeChecked] = React.useState<boolean>(false);
+  const [isHeaderUtilsOpen, setIsHeaderUtilsOpen] = React.useState<boolean>(false);
+  const [selectedHeaderUtils, setSelectedHeaderUtils] = React.useState<string | SelectOption>();
+
+  const headerUtilsOptions = [
+    <SelectOption key={0} value="English" />,
+    <SelectOption key={1} value="Mandarin" />,
+    <SelectOption key={2} value="Hindi" />,
+    <SelectOption key={3} value="Spanish" />,
+    <SelectOption key={4} value="Portuguese" />,
+    <SelectOption key={5} value="Arabic" />,
+    <SelectOption key={6} value="Bengali" />
+  ];
+
+  const onHeaderUtilsToggle = (isExpanded: boolean) => {
+    setIsHeaderUtilsOpen(isExpanded);
+  };
+
+  const onHeaderUtilsSelect = (
+    _event: React.MouseEvent<Element, MouseEvent> | React.ChangeEvent<Element>,
+    value: string | SelectOption
+  ) => {
+    setSelectedHeaderUtils(value);
+    setIsHeaderUtilsOpen(false);
+  };
+
+  const headerUtils = (
+    <Select
+      variant="single"
```

Comment:
Since the default for this prop is already `SelectVariant.single` we could omit this line